indy_node/persistence/state_ts_store.py

```diff
@@ -0,0 +1,41 @@
+from stp_core.common.log import getlogger
+from storage.kv_store_leveldb_int_keys import KeyValueStorageLeveldbIntKeys
+
+logger = getlogger()
+
+
+class StateTsDbStorage():
+    def __init__(self, name, db_dir, db_name):
+        logger.debug("Initializing timestamp-root_hash storage for revocation")
+        self._storage = KeyValueStorageLeveldbIntKeys(db_dir, db_name)
+        self._name = name
+
+    def __repr__(self):
+        return self._name
+
+    def get(self, timestamp: int):
+        value = self._storage.get(str(timestamp))
+        return value
+
+    def set(self, timestamp: int, root_hash: bytes):
+        self._storage.put(str(timestamp), root_hash)
+
+    def close(self):
+        self._storage.close()
+
+    def get_equal_or_prev(self, timestamp):
```

Comment:
I think this needs to be moved to KeyValueStorage Interface level, so that we can easily replace leveldb by rockdsb in this class.